### ISSUE TITLE
Build fails on io.js 1.0.2

### ISSUE THREAD

**Issue by redism** [laverdet/node-fibers#203]
Attaching build log.
iojs/io.js#456

```
> fibers@1.0.4 install /Users/redism/dev/sandbox/node-fibers
> node ./build.js

child_process: customFds option is deprecated, use stdio instead.
child_process: customFds option is deprecated, use stdio instead.
2015-01-17 15:22:11.834 xcodebuild[30043:1007] [MT] PluginLoading: Required plug-in compatibility UUID C4A681B0-4A26-480E-93EC-1218098B9AA0 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Unity4XC.xcplugin' not present in DVTPlugInCompatibilityUUIDs
  CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc:140:3: error: no member named 'SetResourceConstraints' in namespace 'v8'; did you mean simply
      'SetResourceConstraints'?
                v8::SetResourceConstraints(isolate, constraints);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
                SetResourceConstraints
../src/fibers.cc:139:7: note: 'SetResourceConstraints' declared here
        void SetResourceConstraints(Isolate* isolate, ResourceConstraints* constraints) {
             ^
1 error generated.
make: *** [Release/obj.target/fibers/src/fibers.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1038:12)
gyp ERR! System Darwin 13.4.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--release"
gyp ERR! cwd /Users/redism/dev/sandbox/node-fibers
gyp ERR! node -v v1.0.2
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok
Build failed
```

**Comment by jonkoops** [comment]
I can confirm this issue, I am experiencing the same issue.

**Comment by jonkoops** [comment]
Issue #185 is a duplicate of this issue.

**Comment by Nessphoro** [comment]
Fix:
Edit fibers.cc to not use the v8:: prefix for SetResourceConstraints

**Comment by yamalight** [comment]
any chance this'll get fixed?

**Comment by Imote** [comment]
the bug reappeared?

**Comment by yamalight** [comment]
@Imote yep, have exactly same message while trying to install fibers using iojs 1.4.3 (from nvm) on mac os

**Comment by Imote** [comment]
same for me, so i'm on nodejs v0.12 now :p @yamalight

**Comment by fnky** [comment]
I can confirm, this also happens for me.

**Comment by jakemmarsh** [comment]
Having this issue as well, preventing me from switching to io.js.

**Comment by zephraph** [comment]
Ran into this today when trying to install NativeScript as well for iojs 1.5.1

**Comment by priezz** [comment]
Still having the issue with iojs v1.6.1

**Comment by zephraph** [comment]
Just as a heads up, this makes NativeScript impossible to use with iojs.

**Comment by redism** [comment]
Just found forked version here. https://github.com/eiriklv/node-fibers/commit/18a141657463c80c7ada05f8535cecfa0adf09c8 I haven't tried it yet, but it might fix this issue.

**Comment by laverdet** [comment]
Sorry for dragging my feet on this. While looking into this bug I discovered some crazy weirdness going on with the stack guard in node 0.10.x and I didn't want to push out a half fix. While I've isolated the issue, I haven't discovered a fix. node 0.12.x & io.js are fine though so I'm going to forget about it.

Could you guys take a quick peek at the [stack-guard](https://github.com/laverdet/node-fibers/tree/stack-guard) branch and let me know if it works for you. It would be *super* helpful if you guys could post the output of `node test` along with your OS version and CPU architecture.

**Comment by pstadler** [comment]
```
$ node --version
v0.12.0

$ uname -a
Darwin Doppler.local 14.1.0 Darwin Kernel Version 14.1.0: Thu Feb 26 19:26:47 PST 2015; root:xnu-2782.10.73~1/RELEASE_X86_64 x86_64

$ git rev-parse HEAD
198c2f4d8f98404e5ddde84e0a8d8f0d21832648

$ node test
already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: *fail*
code: 0
stderr:
stdout:
started.js: pass
unwind.js: pass
```

**Comment by yamalight** [comment]
@laverdet just tried running tests in stack-guard branch with latest iojs on my machine, all seem to pass:
```
➜  node-fibers git:(stack-guard) ✗ node -v
v1.6.2
➜  node-fibers git:(stack-guard) ✗ uname -a
Darwin Tims-MacBook-Pro.local 14.1.0 Darwin Kernel Version 14.1.0: Thu Feb 26 19:26:47 PST 2015; root:xnu-2782.10.73~1/RELEASE_X86_64 x86_64
➜  node-fibers git:(stack-guard) ✗ git rev-parse HEAD
198c2f4d8f98404e5ddde84e0a8d8f0d21832648
➜  node-fibers git:(stack-guard) ✗ node test
already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: pass
started.js: pass
unwind.js: pass
```

**Comment by redism** [comment]
test-1
```
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node --version
v0.10.29
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: uname -a
Darwin Jeffs-MacBook-Pro.local 14.1.0 Darwin Kernel Version 14.1.0: Thu Feb 26 19:26:47 PST 2015; root:xnu-2782.10.73~1/RELEASE_X86_64 x86_64
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: git rev-parse HEAD
198c2f4d8f98404e5ddde84e0a8d8f0d21832648
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node test
already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: *fail*
code: 0
stderr:
stdout:
started.js: pass
unwind.js: pass
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers:
```

test-2
```
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node --version
v0.12.1
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node test
already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: *fail*
code: null
stderr:
stdout:
started.js: pass
unwind.js: pass
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers:
```
test-3
```
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node --version
v1.6.2
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers: node test
already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: pass
started.js: pass
unwind.js: pass
[-:(detached from origin/stack-guard)] redism@~/dev/sandbox/node-fibers:
```

**Comment by priezz** [comment]
Please advice, what am I doing wrong...
```
%node --version
v1.6.2
```
```
%uname -a
Darwin macbook.home 14.3.0 Darwin Kernel Version 14.3.0: Tue Mar 10 20:10:46 PDT 2015; root:xnu-2782.20.44.1.1~1/RELEASE_X86_64 x86_64
```

```
%git clone --branch stack-guard https://github.com/laverdet/node-fibers.git`
%cd node-fibers/
%npm install -g
```
gives the same `child_process: customFds option is deprecated, use stdio instead`
`%npm test` gives multiple `... throw new Error(''+ modPath+ '.node is missing. Try reinstalling node-fibe...`

`npm install -g "laverdet/node-fibers#stack-guard"` returns same errors as installing from cloned dir.

**Comment by laverdet** [comment]
@pstadler & @redism interesting that you're getting failures in node 0.12.x. I'm running the same machine configuration (OS X, x64) and mine is fine. This is with a fresh build of fibers, right?

Could you try modifying `src/fibers.cc` line 594 and change the `1024 * 4` to `1024 * 8` and see if that works? Run `node build -f` to rebuild the extension. If that doesn't work I have no idea lmao

**Comment by priezz** [comment]
Just to report that Ifixed the problem. Currently `node test` passes all the tests.

**Comment by redism** [comment]
@laverdet I've tested as you told, but it still failed. But after changing ```1024 * 4``` to ```1024 * 30```, test passes. So I tried to check the minimum number that passes the test, and ```1024 * 13``` passes.

Hope this helps.

**Comment by emgee3** [comment]
For what it's worth, I have issues with node 0.12 failing two stack overflow tests:

    mg@iMac:~/Code/node-fibers$ uname -a
    Darwin iMac.local 14.3.0 Darwin Kernel Version 14.3.0: Mon Mar 16 23:12:31 PDT 2015; root:xnu-2782.20.48~4/RELEASE_X86_64 x86_64
    mg@iMac:~/Code/node-fibers$ nvm use 0.12
    Now using node v0.12.1
    mg@iMac:~/Code/node-fibers$ node -v
    v0.12.1
    mg@iMac:~/Code/node-fibers$ npm -v
    2.5.1
    mg@iMac:~/Code/node-fibers$ git branch
      master
    * stack-guard
    mg@iMac:~/Code/node-fibers$ npm install

    > fibers@1.0.5 install /Users/mg/Code/node-fibers
    > node ./build.js

    child_process: customFds option is deprecated, use stdio instead.
    child_process: customFds option is deprecated, use stdio instead.
      CXX(target) Release/obj.target/fibers/src/fibers.o
      CXX(target) Release/obj.target/fibers/src/coroutine.o
      CC(target) Release/obj.target/fibers/src/libcoro/coro.o
      SOLINK_MODULE(target) Release/fibers.node
      SOLINK_MODULE(target) Release/fibers.node: Finished
    Installed in `/Users/mg/Code/node-fibers/bin/darwin-x64-v8-3.28/fibers.node`
    mg@iMac:~/Code/node-fibers$ npm test

    > fibers@1.0.5 test /Users/mg/Code/node-fibers
    > node ./test.js

    already-running.js: pass
    bad-context.js: pass
    child-process.js: pass
    current.js: pass
    exec.js: pass
    exit.js: pass
    fibonacci.js: pass
    finish-multiple.js: pass
    future-exception.js: pass
    future.js: pass
    illegal-yield.js: pass
    pool.js: pass
    process-title.js: pass
    stack-overflow.js: pass
    stack-overflow2.js: *fail*
    code: null
    stderr: 
    stdout: 
    stack-overflow3.js: *fail*
    code: null
    stderr: 
    stdout: 
    started.js: pass
    unwind.js: pass
    mg@iMac:~/Code/node-fibers$ 

However it works fine in io.js:

    mg@iMac:~/Code/node-fibers$ nvm use iojs
    Now using io.js v1.6.2
    mg@iMac:~/Code/node-fibers$ node -v
    v1.6.2
    mg@iMac:~/Code/node-fibers$ npm -v
    2.7.1
    mg@iMac:~/Code/node-fibers$ npm install

    > fibers@1.0.5 install /Users/mg/Code/node-fibers
    > node ./build.js

    child_process: customFds option is deprecated, use stdio instead.
    child_process: customFds option is deprecated, use stdio instead.
      CXX(target) Release/obj.target/fibers/src/fibers.o
      CXX(target) Release/obj.target/fibers/src/coroutine.o
      CC(target) Release/obj.target/fibers/src/libcoro/coro.o
      SOLINK_MODULE(target) Release/fibers.node
      SOLINK_MODULE(target) Release/fibers.node: Finished
    Installed in `/Users/mg/Code/node-fibers/bin/darwin-x64-v8-4.1/fibers.node`
    mg@iMac:~/Code/node-fibers$ npm test

    > fibers@1.0.5 test /Users/mg/Code/node-fibers
    > node ./test.js

    already-running.js: pass
    bad-context.js: pass
    child-process.js: pass
    current.js: pass
    exec.js: pass
    exit.js: pass
    fibonacci.js: pass
    finish-multiple.js: pass
    future-exception.js: pass
    future.js: pass
    illegal-yield.js: pass
    pool.js: pass
    process-title.js: pass
    stack-overflow.js: pass
    stack-overflow2.js: pass
    stack-overflow3.js: pass
    started.js: pass
    unwind.js: pass
    mg@iMac:~/Code/node-fibers$

**Comment by yelworc** [comment]
FWIW, I seem to get different results from everybody else, in that `stack-overflow2.js` fails but `stack-overflow3.js` passes with io.js:

	vagrant@vagrant:/vagrant/3rdparty/node-fibers$ uname -a
	Linux vagrant 3.2.0-4-amd64 #1 SMP Debian 3.2.65-1+deb7u2 x86_64 GNU/Linux
	vagrant@vagrant:/vagrant/3rdparty/node-fibers$ node --version
	v1.6.3
	vagrant@vagrant:/vagrant/3rdparty/node-fibers$ node test
	already-running.js: pass
	bad-context.js: pass
	child-process.js: pass
	current.js: pass
	exec.js: pass
	exit.js: pass
	fibonacci.js: pass
	finish-multiple.js: pass
	future-exception.js: pass
	future.js: pass
	illegal-yield.js: pass
	pool.js: pass
	process-title.js: pass
	stack-overflow.js: pass
	stack-overflow2.js: *fail*
	code: null
	stderr:
	stdout:
	stack-overflow3.js: pass
	started.js: pass
	unwind.js: pass

Same result with io.js v1.6.2. With node v0.12.2, both `stack-overflow2.js` and `stack-overflow3.js` are failing for me.

**Comment by mirageglobe** [comment]
Just a quick note. 

v0.12.2 -> works fine; except warnings
child_process: customFds option is deprecated, use stdio instead.
child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/fibers/src/fibers.o
  CXX(target) Release/obj.target/fibers/src/coroutine.o
  CC(target) Release/obj.target/fibers/src/libcoro/coro.o
  SOLINK_MODULE(target) Release/fibers.node
  SOLINK_MODULE(target) Release/fibers.node: Finished

iojs-v1.7.1 -> fails on fibers.o
CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc:140:3: error: no member named 'SetResourceConstraints' in namespace 'v8'; did you mean simply 'SetResourceConstraints'?
                v8::SetResourceConstraints(isolate, constraints);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
                SetResourceConstraints
../src/fibers.cc:139:7: note: 'SetResourceConstraints' declared here
        void SetResourceConstraints(Isolate* isolate, ResourceConstraints* constraints) {
             ^
1 error generated.

**Comment by ChrisAlvares** [comment]
On OSX 10.10 x64 iojs v1.8.1

$ npm test
> fibers@1.0.5 test node_modules/node-fibers
> node ./test.js

already-running.js: pass
bad-context.js: pass
child-process.js: pass
current.js: pass
exec.js: pass
exit.js: pass
fibonacci.js: pass
finish-multiple.js: pass
future-exception.js: pass
future.js: pass
illegal-yield.js: pass
pool.js: pass
process-title.js: pass
stack-overflow.js: pass
stack-overflow2.js: pass
stack-overflow3.js: pass
started.js: pass
unwind.js: pass

any update on this @laverdet

**Comment by metamatt** [comment]
The stack-guard branch is already merged to master so I'm guessing some of the experiments here are no longer necessary.

I tried node-fibers 1.0.5 with the current iojs release (2.1.0) and found a different compile-time problem; see #227.

**Comment by pstadler** [comment]
Any idea when a new version containing the changes merged with stack-guard is gonna be released?

**Comment by laverdet** [comment]
Fibers should compile just fine on iojs at this point.

**Comment by awsp** [comment]
@laverdet 
I am still getting OP's issue, under io.js version 3.1.0, but got no problem compiling after switching back to node.js 0.12.7 on Mac OS X 10.10.5.

**Comment by jpmelnik** [comment]
Any solution for this? I can't install fibers and I can't deploy meteor app.

node -v    => v4.2.1
nodejs -v => v0.12.7
Meteor 1.2.1

**Comment by sjmcdowall** [comment]
@jpmelnik -- The only supported node version for meteor is 0.10.40 for Meteor 1.2.1. It also contains (the meteor package) a forked version of fibers especially for Meteor .. you should (need) to be using those versions on any meteor deployment I would think.

**Comment by jpmelnik** [comment]
Ok, I changed the version of Node and I get the same issue:

---------

> fibers@1.0.5 install /home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers
> node ./build.js

(node) child_process: options.customFds option is deprecated. Use options.stdio instead.
gyp WARN EACCES user "root" does not have permission to access the dev dir "/root/.node-gyp/4.2.1"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/.node-gyp"
make: se ingresa al directorio «/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/build»
  CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc: In function ‘v8::Handle<v8::Signature> uni::NewSignature(v8::Isolate*, v8::Handle<v8::FunctionTemplate>, int, v8::Handle<v8::FunctionTemplate>*)’:
../src/fibers.cc:132:54: error: no matching function for call to ‘v8::Signature::New(v8::Isolate*&, v8::Handle<v8::FunctionTemplate>&, int&, v8::Handle<v8::FunctionTemplate>*&)’
   return Signature::New(isolate, receiver, argc, argv);
                                                      ^
../src/fibers.cc:132:54: note: candidate is:
In file included from /home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/.node-gyp/4.2.1/include/node/node.h:42:0,
                 from ../src/coroutine.h:1,
                 from ../src/fibers.cc:1:
/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/.node-gyp/4.2.1/include/node/v8.h:4675:27: note: static v8::Local<v8::Signature> v8::Signature::New(v8::Isolate*, v8::Local<v8::FunctionTemplate>)
   static Local<Signature> New(
                           ^
/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/.node-gyp/4.2.1/include/node/v8.h:4675:27: note:   candidate expects 2 arguments, 4 provided
../src/fibers.cc: In function ‘void uni::SetResourceConstraints(v8::Isolate*, v8::ResourceConstraints*)’:
../src/fibers.cc:140:3: error: ‘SetResourceConstraints’ is not a member of ‘v8’
   v8::SetResourceConstraints(isolate, constraints);
   ^
../src/fibers.cc:140:3: note: suggested alternative:
../src/fibers.cc:139:7: note:   ‘uni::SetResourceConstraints’
  void SetResourceConstraints(Isolate* isolate, ResourceConstraints* constraints) {
       ^
../src/fibers.cc: In function ‘v8::Handle<v8::Signature> uni::NewSignature(v8::Isolate*, v8::Handle<v8::FunctionTemplate>, int, v8::Handle<v8::FunctionTemplate>*)’:
../src/fibers.cc:133:2: warning: control reaches end of non-void function [-Wreturn-type]
  }
  ^
make: *** [Release/obj.target/fibers/src/fibers.o] Error 1
make: se sale del directorio «/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/build»
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 3.16.0-51-generic
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--release"
gyp ERR! cwd /home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers
gyp ERR! node -v v4.2.1
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
Build failed
npm ERR! Linux 3.16.0-51-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE

npm ERR! fibers@1.0.5 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fibers@1.0.5 install script 'node ./build.js'.
npm ERR! This is most likely a problem with the fibers package,
npm ERR! not with npm itself.

**Comment by jpmelnik** [comment]
I get new issue:

> fibers@1.0.5 install /home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers
> node ./build.js

(node) child_process: options.customFds option is deprecated. Use options.stdio instead.
internal/child_process.js:298
    throw errnoException(err, 'spawn');
    ^

Error: spawn EACCES
    at exports._errnoException (util.js:874:11)
    at ChildProcess.spawn (internal/child_process.js:298:11)
    at Object.exports.spawn (child_process.js:339:9)
    at build (/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/build.js:55:5)
    at Object.<anonymous> (/home/jpmelnik/prueba/bundle/programs/server/node_modules/fibers/build.js:47:3)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)

npm ERR! fibers@1.0.5 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fibers@1.0.5 install script.
npm ERR! This is most likely a problem with the fibers package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls fibers
npm ERR! There is likely additional logging output above.
npm ERR! System Linux 3.16.0-51-generic
npm ERR! command "/root/.nvm/v0.10.40/bin/node" "/root/.nvm/v0.10.40/bin/npm" "install"
npm ERR! cwd /home/jpmelnik/prueba/bundle/programs/server
npm ERR! node -v v0.10.40
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0

**Comment by tolgap** [comment]
@jpmelnik I have the same issue right now, same versions as you. What did you do to fix it?

**Comment by laverdet** [comment]
You're having an issue compiling fibers on an abandoned version of iojs from over a year ago?

**Comment by tolgap** [comment]
Not iojs, I said nodejs. I mentoined jpmelnik in my comment (he listed his versions above). And it is really not that weird because you could be coming back to a very old project only to run into issues when you updated something else (which is my case right now).

Anyway, I found the fix somewhere else. You need the exact version of node.js being `v0.10.40`/

**Comment by laverdet** [comment]
Ahh, was this under a previous version of Meteor then?

Fibers 1.0.5 is really old so if your error message says 1.0.5 you should consider updating from npm.

**Comment by sjmcdowall** [comment]
@tolgap -- Is this also a Meteor app?  If so what version and how are you running it.  Prior to Meteor 1.4 (and I have no idea if its still true) -- MDC included their own forked version of Fibers (1.0.5) without (alas) much binary support so moving to almost any production environment meant you had to have a full development environment to compile their version of Fibers.  

If this is 1.4+ (and I hope it is) I don't know what version of Fibers Meteor now includes in a new project, etc. or even if the hacked Fibers is still needed or if the stock one is ok.  You may want to ask the Meteor forum.  Of course if this is NOT Meteor .. never mind. :)

**Comment by tolgap** [comment]
@sjmcdowall Yes this is a meteor app. I do a manual deployment. It's running version 1.1.0.2. But the fix was (as said above) that I was using the wrong version of node/npm.
